### DISCUSSION TITLE
S3 Hosting contents don't display

### DISCUSSION THREAD

**Discussion by cardamom-ai** [jupyterlite/jupyterlite#1063] [Q&A]
Hi all,
    I'm working on hosting a jupyterlite notebook in S3 as a static site for a demo.

    I've got the static site loading, but the homepage doesn't display any of the contents I've included in the build.

    I noticed that if I load the notebook via URL and include the full bucket URL everything appears as expected, so I think I'm overlooking something small or just missing some glue.

    Incidentally if you all would like me to add some documentation on how to host via S3 or build a pipeline as with the Github Pages documentation I'd be willing to.

**Comment by jtpio** [comment]
Thanks @cardamom-ai.

> Incidentally if you all would like me to add some documentation on how to host via S3 or build a pipeline as with the Github Pages documentation I'd be willing to.

That would be great! Which approach are you using? Creating a new Drive that gets added to the left area?

Also feel free to share any screenshot or errors in the dev tools console if there are any, that could help troubleshoot the issue.
